aveas_openlabel/metadata.py

```diff
@@ -89,6 +89,12 @@ class Metadata(BaseMetadata):
     - Z: 'Z' if the time zone is UTC, '±[hh]:[mm]', '±[hh][mm]', or '±[hh]' otherwise, ex. '+0100'
     """
 
+    opendrive: str = field(default_factory=lambda: no_default(field="Metadata.opendrive"), metadata=required)
+    """
+    The name of the OpenDRIVE file belonging to the OpenLABEL, i.e., OpenDRIVE related specifications such as the lane or road ID refer to this OpenDRIVE file. 
```

Comment:
```suggestion
    The name of the OpenDRIVE file which contains specifications that fields in this OpenLABEL refer to. 
```

I'm a bit confused by the sentence structure and grammar. I would like the description to be more straightforward. Is the proposed explanation correct or did I misunderstand?

Reply:
changes as requested have been applied, ruff, black, and mypy rund successfully